frontend/src/content/news.json

```
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-30",
```

The date should be updated to today:
```suggestion
      "date": "2022-08-12",
```